### DISCUSSION TITLE
Spotify Plugin returns ModuleNotFound Error

### DISCUSSION THREAD

**Discussion by fetkitty** [beetbox/beets#4183] [Q&A] [answered]
After adding the following to my config.yaml:

```
plugins: [spotify] 
pluginpath: C:\Users\*Name*\AppData\Local\Programs\Python\Python310\Lib\site-packages\beetsplug\spotify.py

spotify:
   source_weight: 0.5
```

beet returns the following:

`beet version`
```
** error loading plugin spotify:
Traceback (most recent call last):
  File "C:\Users\*Name*\AppData\Local\Programs\Python\Python310\lib\site-packages\beets\plugins.py", line 268, in load_plugins
    namespace = __import__(modname, None, None)
  File "C:\Users\*Name*\AppData\Local\Programs\Python\Python310\lib\site-packages\beetsplug\spotify.py", line 26, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'


beets version 1.6.0
Python version 3.10.0
no plugins loaded
```
i've tried installing the spotify plugin using `pip install beets[spotify]`

my apologies if i forgot to add anything important, i'm not very experienced at coding or even at using the command line for that matter.

**Comment by sampsyo** [comment, marked as answer]
You should be able to fix this with something like `pip install requests`. We do not have a `[spotify]` extra yet, so `pip install beets[spotify]` won't do anything more than `pip install beets`:
https://github.com/beetbox/beets/blob/f315a13a1bb670f5b35a9fd43c450acec28621a8/setup.py#L123-L145

**Reply by fetkitty** [reply]
Thank you so much for the quick reply! That fixed it for me :). I'm gonna go ahead and mark this as solved.
